plugin/src/main/java/io/jenkins/plugins/remotingkafka/GlobalKafkaConfiguration.java

```diff
@@ -211,6 +225,44 @@ public FormValidation doTestBrokerConnection(@QueryParameter("brokerURL") final
         }
     }
 
+    @RequirePOST
+    public FormValidation doTestKubernetesConnection(
+            @QueryParameter("kubernetesUrl") String serverUrl,
+            @QueryParameter("kubernetesCredentialsId") String credentialsId,
+            @QueryParameter("kubernetesCertificate") String serverCertificate,
+            @QueryParameter("kubernetesSkipTlsVerify") boolean skipTlsVerify,
+            @QueryParameter("kubernetesNamespace") String namespace
+    ) {
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+
+        try (KubernetesClient client = new KubernetesFactoryAdapter(serverUrl, namespace,
+                Util.fixEmpty(serverCertificate), Util.fixEmpty(credentialsId), skipTlsVerify
+        ).createClient()) {
+            client.pods().list();
+            return FormValidation.ok("Success");
+        } catch (KubernetesClientException e) {
+            LOGGER.log(Level.FINE, String.format("Error testing Kubernetes connection %s", serverUrl), e);
```

Comment:
I think we should use `Level.SEVERE` for Exception like this

Reply:
I don't think that's the case. Because `doTestKubernetesConnection` function is optionally triggered by users and we will fully report the result back to user via `FormValidation`. We catch exceptions because we want to notify users that the connection has failed, they're not internal server errors.